web/spec/supabase.yml

```diff
@@ -576,6 +576,22 @@ pages:
             });
           });
           ```
+      - name: Sign in with Open ID Connect.
+        description: |
+          Supabase support signing in with Open ID Connect.
+        js: |
+          ```js
+           const { user, session, error } = await supabase.auth.signIn({
+            phone: '+13334445555',
+            password: 'some-password',
```

Comment:
you can remove password and phone fields for this snippet

Reply:
I think?

Reply:
yep, thanks for the catch!